lib/src/browser/disposable_browser.dart

```diff
@@ -21,18 +21,14 @@ import 'package:w_common/func.dart';
 import 'package:w_common/src/common/disposable.dart' as disposable_common;
 
 class _InnerDisposable extends disposable_common.Disposable {
-  Func<Future<Null>> onDisposeHandler;
-  Func<Future<Null>> onWillDisposeHandler;
+  Func<Future<void>> onDisposeHandler;
+  Func<Future<void>> onWillDisposeHandler;
```

Comment:
`void` seems more reasonable when we don't return anything, to me.

Reply:
Is this syntax supported in Dart 1.x?

Reply:
I'm not sure. I am focusing on Dart 2, I'll revert whatever doesn't work in Dart 1 if necessary.